cloudant-client/src/main/java/com/cloudant/client/api/views/ViewResponse.java

```diff
@@ -119,6 +119,13 @@
          * @since 2.10.0
          */
         String getError();
+
+        /**
+         * Returns a "sparse" document consisting of only the document ID, the revision ID, and the
+         * deleted flag
+         * @return a sparse document
+         */
+        Document getSparseDocument();
```

Comment:
I'm not sure sparse is the best word to identify these fields but my idea of `getPartialDeletedDoc` or `getIdRevDeletedFields` might not be much better.